.github/workflows/conda-cpp-tests.yaml

```diff
@@ -62,37 +59,25 @@ jobs:
 
           export MATRICES="
             pull-request:
-              - { CUDA_VER: '11.2.2', LINUX_VER: 'centos7', ARCH: 'amd64', PY_VER: '3.9', GPU: 'v100', DRIVER: 'earliest' }
```

Comment:
Dropping this means no `centos7` or `earliest` driver coverage in PRs. Might want the `11.4` to use those?

Reply:
I'm good with that.

Reply:
Done.